src/metal/window-proxy.ts

```diff
@@ -40,6 +42,7 @@ let W: WindowProxy = {
   getScrollLeft: nop,
   getHeight: nop,
   getWidth: nop,
+  onWindowIsDirtyListeners: [],
```

Comment:
do we need to delete stuff from this array to avoid memory leaks?

Reply:
yeah it can't hurt. I'll include a private cleanup method within the existing `spaniel-observer` class.